scripts/limonade/Dockerfile

```diff
@@ -56,10 +46,7 @@ ENV NODE_ENV production
 # Run the application as a non-root user.
 USER node
 
-# Copy package.json so that package manager commands can be used.
-COPY package.json .
-
 # Copy the production dependencies from the deps stage and also
 # the built application from the build stage into the image.
-COPY --from=deps /usr/src/app/node_modules ./node_modules
-COPY --from=build /usr/src/app/dist ./dist
+COPY --from=build /prod/limonade/node_modules ./node_modules
+COPY --from=build /usr/src/app/scripts/limonade/dist ./dist
```

Comment:
If you create a parameterized dockerfile then you wouldn't have to duplicate this, again see https://github.com/pyth-network/pyth-crosschain/blob/main/Dockerfile.node and how I run it with e.g. https://github.com/pyth-network/pyth-crosschain/blob/main/.github/workflows/push-xc-admin-images.yml#L31

Reply:
Tnx for this comment. Now using a shared dockerfile for node services